compiler/noirc_driver/src/lib.rs

```diff
@@ -445,8 +445,8 @@ pub fn compile_main(
     }
 
     if options.print_acir {
-        println!("Compiled ACIR for main (unoptimized):");
-        println!("{}", compiled_program.program);
+        noirc_errors::print_to_stdout(format_args!("Compiled ACIR for main (unoptimized):\n"));
+        noirc_errors::print_to_stdout(format_args!("{}\n", compiled_program.program));
```

Comment:
It would be nice if this was a bit less verbose, and that we wouldn't have to remember to put a `\n` in it.

Reply:
Done